sensors.c

```diff
@@ -784,14 +784,25 @@ int status_proc(int msg, DIALOG *d, int c)
          
             if (comport.status == READY)
             {
-               if (device_connected)
-                  sprintf(d->dp, " COM%i ready (device connected)", comport.number + 1);
-               else
-                  sprintf(d->dp, " COM%i ready (device not found)", comport.number + 1);
+               if (device_connected) {
```

Comment:
Similarly, I'd rather this reported actual device paths.

Reply:
Also, in a future major version, the user interface might be updated accordingly to better support USB, PTY and other interfaces. This is just a simple patch on the current version.

Reply:
Sure. I'm saying that I think the minimum needed to support this feature is to report the correct thing in the existing places.

Reply:
I'm not sure I understood what you mean. Can you please better explain which code modification is needed and where? Thanks